canonical/grafana-agent-k8s-operator
# add support for the grafana cloud integrator charm

## Issue
Some users don't want to run their own COS Lite, or would like to run it in tandem with Grafana Cloud.


## Solution
Integrate Grafana Agent with the Grafana Cloud Integrator Charm.
Closes #153.

## Context

N/A


## Testing Instructions

1. Deploy Grafana Agent
2. Deploy Grafana Cloud Integrator
3. Configure Grafana Cloud Integrator with your cloud credentials and URLs
4. Relate the integrator to the agent
5. Validate in Grafana Cloud that metrics and logs are coming in as expected.

## Release Notes

Add support for Grafana Cloud
